documentation/src/main/asciidoc/stories/assembly_integrating.adoc

```diff
@@ -4,7 +4,7 @@
 include::{topics}/integrations.adoc[leveloffset=+0]
 //topic includes the following:
 //include::{topics}/integrations_hibernate_cache.adoc[leveloffset=+2]
-//include::{topics}/integrations_spring.adoc[leveloffset=+2]
+include::{topics}/integrations_spring.adoc[leveloffset=+0]
```

Comment:
can we make this consistent? or `integrations.adoc` includes both "hibernate" and "spring" docs or nothing... 
do you agree?